add access log forwarding to stdout

dmfine opened the pull request:
So that it will be recorded on the docker host and forwarded to papertrail.

Commit:
add access log forwarding to stdout so that papertrail will pick it up on the docker host

dmfine commented:
The previous PR had the access_log directive at the top level, which it turns out is not allowed. Nginx starts when it's under the server block. QA has some issue at the moment which is preventing me from testing it as thoroughly as I'd like.

seanh commented:
The commit was based off an old, broken version of the master branch. I've rebased it for you

seanh commented on conf/nginx.conf:
```diff
@@ -33,7 +33,7 @@ http {
         listen 9083;
         merge_slashes off;
-
+        access_log /dev/stdout;
```
What's the difference between putting this in the `server { ... }` block, vs putting it higher up in the `http { ... }` block? It looks like putting it in the `server` block might mean that it applies only to that one virtual server, when the `nginx.conf` might contain multiple `server`'s in the future.

seanh replied:
Yeah, this should be in the `http { ... }`. See https://www.nginx.com/resources/wiki/start/topics/examples/full/ and https://nginx.org/en/docs/http/ngx_http_log_module.html#access_log. I've pushed a commit to fix it

dmfine replied:
future proofing +1

seanh committed:
Move access_log into http { ... }

Move NGINX's access_log setting into the parent http { ... } block where
it will apply to all servers, rather than having it in one particular
server { ... } block where it only applies to that one virtual server.
